Formula/vert.x.rb

```diff
@@ -1,8 +1,8 @@
 class VertX < Formula
   desc "Toolkit for building reactive applications on the JVM."
   homepage "http://vertx.io/"
-  url "https://dl.bintray.com/vertx/downloads/vert.x-3.3.3-full.tar.gz"
-  sha256 "9f7d374ad10726af12a203aa089e0f4e5cc68d8d1a82bb71f567d9f75005e99a"
+  url "https://dl.bintray.com/vertx/downloads/vert.x-3.4.0-full.tar.gz"
+  sha256 "dfd8bacc19c6a0c40c98cf557eabb688692ead0be749ba2f1200505eb490d3b0"
 
   bottle :unneeded
 
```

Comment:
Can you add a `depends_on :java` line? (or `depends_on :java => "1.8+"` if there is a java version requirement). It should go after the `bottle :unneeded` line.